nautobot/extras/webhooks.py

```diff
@@ -2,10 +2,44 @@
 from django.utils import timezone
 
 from nautobot.utilities.api import get_serializer_for_model
-from nautobot.extras.models import Webhook
+from nautobot.extras.models import Webhook, ObjectChange
 from nautobot.extras.registry import registry
 from nautobot.extras.tasks import process_webhook
 from .choices import ObjectChangeActionChoices
+from ..utilities.utils import shallow_compare_dict
+
+
+def get_snapshots(instance, action):
+    prechange = None
+
+    if action != ObjectChangeActionChoices.ACTION_CREATE:
+        changed_object_type = ContentType.objects.get_for_model(instance)
+        object_datav2 = ObjectChange.objects.filter(
+            changed_object_type=changed_object_type, changed_object_id=instance.id
+        )[:2]
+        prechange = object_datav2[1].object_datav2
+
+    serializer_class = get_serializer_for_model(instance.__class__)
+
+    postchange = (
+        serializer_class(instance, context={"request": None}).data
+        if action != ObjectChangeActionChoices.ACTION_DELETE
+        else None
+    )
+
+    if prechange and postchange:
+        diff_added = shallow_compare_dict(prechange, postchange, exclude=["last_updated"])
+        diff_removed = {x: prechange.get(x) for x in diff_added}
+    elif prechange and not postchange:
+        diff_added, diff_removed = None, prechange
+    else:
+        diff_added, diff_removed = postchange, None
+
+    return {
+        "prechange": prechange,
+        "postchange": postchange,
+        "differences": {"removed": diff_removed, "added": diff_added},
+    }
```

Comment:
Would you mind taking a pass at updating `nautobot/docs/models/extras/webhook.md` to account for the changes made in this PR?